### DISCUSSION TITLE
Feature Request: Driver Tracker Opacity

### DISCUSSION THREAD

**Discussion by NeuroDawg** [f1multiviewer/issue-tracker#230] [Feature requests] [answered]
Another feature request, if possible.

I the Driver Tracker is not an F1 stream (DRM restricted), can there be an opacity setting for the background to allow us to overlay it over the main race window (or any other window that is open)?

I'd love to be able to make the background opaque so that I could put it in the upper right corner of my main race window so that I can see a track map/driver position at all times during the race.

**Comment by f1multiviewer** [comment, marked as answer]
Coming soon!
![track-map-transparency](https://user-images.githubusercontent.com/106821994/193433118-22c0531b-6e56-4f7d-a4e8-1128294a7d78.jpg)
(this is just a transparent window dragged on top of another window)

**Comment by xiphiasphotography** [comment]
Can this be a setting. This opaque track map is realy anoying since I have it on an extra screen. Make it a option to have it opaque.

**Reply by f1multiviewer** [reply]
You can, you can press "T" or change it in settings.
